examples/cascadeproxy-socks/README.md

```diff
@@ -0,0 +1,29 @@
+# CascadeSocksProxy
+
+`CascadeSocksProxy` is an example that shows an aggregator server that forwards
+the requests to another socks proxy server. This example is written base on `cascadeproxy` example.
+
+Diagram:
+```
+client --> goproxy --> socks5 proxy --> internet
+```
+
+This example starts a HTTP/HTTPS proxy using goproxy that listens on port `8080`, and forward the requests to the socks5 proxy on `socks5://localhost:1080`.
+
+

```

Comment:
Can you mention the fact that we do MITM to use the SOCKS proxy and that the proxy forwarding is impemented by the Go standard library?